### ISSUE TITLE
I have a question about the way you wanted to implement.

### ISSUE THREAD

**Issue by kye9216789** [open-mmlab/mmdetection#364]
Hi.
I have been searched about relation between meaning of validation loss and evaluation result(mAP).
But I failed to find concrete solution.
I found your reply : 
https://github.com/open-mmlab/mmdetection/issues/171#issuecomment-446434197

My question is, Why you choose the second option? My expectancy is,
First : First and second options have exactly same meaning
Second : First option is pointless to enhance training result
Third : Second option is easy to implement and use
Fourth : Ordinarily Object detection methods are using just second option, not first option.

I can not sleep because of my curiosity about this question. 
I need your help.

Thank you.

**Comment by hellock** [comment]
Unlike classification, the loss and performance (mAP) does not mean exactly the same thing for object detection.
You can choose both ways for validation, and we evaluate the mAP by default.
If you don't specify `--validate` and use two phases in the `workflow` field, then it will just calculate the val loss instead of computing mAP. It will be faster.

**Comment by kye9216789** [comment]
You mean, when loss is very low, the performance may not good in object detection?
I cannot understand. 
Can you recommend me any document or paper?
Is it your experience?

I always thank for your brilliant sight and clear answer.

**Comment by hust-kevin** [comment]
> You mean, when loss is very low, the performance may not good in object detection?
> I cannot understand.
> Can you recommend me any document or paper?
> Is it your experience?
> 
> I always thank for your brilliant sight and clear answer.

hi, do you understand the the relation of loss and ap?
